src/sources/video_card.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Credits: https://stackoverflow.com/a/49267844
+ */
+
+export interface VideoCard {
+  vendor: string
+  renderer: string
+}
+
+export default function getVideoCard(): VideoCard | undefined {
+  const gl = document.createElement('canvas').getContext('webgl')
```

Comment:
Some old browsers that we support don't support `'webgl'`. So please use the following algorithm:

1. Try creating a context with `'webgl'`
2. If the returned value is not an object, try creating a context with `'experimental-webgl'`
3. The rest is the same